trailblazer/server/api.py

```diff
@@ -85,3 +94,80 @@ def aggregate_jobs():
 
     data = store.aggregate_failed(one_month_ago)
     return jsonify(jobs=data)
+
+
+# CG REST INTERFACE ###
+
+
+@blueprint.route("/query-analyses", methods=["POST"])
+def post_query_analyses():
+    content = request.json
+    query_analyses = store.analyses(
+        case_id=content.get("case_id"),
+        query=content.get("query"),
+        status=content.get("status"),
+        deleted=content.get("deleted"),
+        temp=content.get("temp"),
+        before=parse_datestr(content.get("before")) if content.get("before") else None,
+        is_visible=content.get("visible"),
+        family=content.get("family"),
+    )
+    data = [stringify_timestamps(analysis_obj.to_dict()) for analysis_obj in query_analyses]
+    return jsonify(*data), 200
+
+
+@blueprint.route("/get-latest-analysis", methods=["POST"])
+def post_get_latest_analysis():
+    content = request.json
+    analysis_obj = store.get_latest_analysis(case_id=content.get("case_id"))
+    if analysis_obj:
+        data = stringify_timestamps(analysis_obj.to_dict())
+        return jsonify(**data), 200
+    return jsonify(None), 200
+
+
+@blueprint.route("/find-analysis", methods=["POST"])
```

Comment:
now, with this method I can find analyses by case_id, started_at and status, right?

Reply:
Yes!

Reply:
All arguments are required.